src/driver/DriverUtils.ts

```diff
@@ -41,7 +41,9 @@ export class DriverUtils {
     }
 
     static isPostgresFamily(driver: Driver): boolean {
-        return ["postgres", "aurora-postgres"].includes(driver.options.type)
+        return ["postgres", "aurora-postgres", "cockroachdb"].includes(
```

Comment:
@AlexMesser need to ensure all the previous `isPostgresFamily` calls work fine for the newly added driver.